### DISCUSSION TITLE
How to make a custom theme?

### DISCUSSION THREAD

**Discussion by T08OG** [orestbida/cookieconsent#582] [Q&A]
I want to include my brand colours into the cookie consent banner, but my knowledge on writing in code is very limited. I saw this threat: src/scss/abstracts/_light-color-scheme.scss but I don't know how to customize this and add it to my HTML-code.

Can someone help me?

**Comment by dombyte** [comment]
> I want to include my brand colours into the cookie consent banner, but my knowledge on writing in code is very limited. I saw this threat: src/scss/abstracts/_light-color-scheme.scss but I don't know how to customize this and add it to my HTML-code.
> 
> 
> 
> Can someone help me?

When you modify the src you need to build the project again to get the css files. 
A simpler and maybe dirtier way, which I also did is to just modify the final css. There you can find the already existing themes which you can use and modify to your likings. That's pretty easy you just need to change the color values for buttons etc. 
When you finished with your css then just load it instead of the "old" one. 
There are explanations on how to implement this plugin for example load it via CDN. You just need to use your file link instead of the CDN CSS file link.
